acceptance-tests/src/test/scala/ch/datascience/graph/acceptancetests/tooling/RDFStore.scala

```diff
@@ -105,8 +134,8 @@ object RDFStore {
                     |PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
                     |
                     |SELECT (COUNT(*) as ?Triples) WHERE { 
-                    |  ?agentS rdf:type <http://purl.org/dc/terms/SoftwareAgent> .
-                    |  ?agentS rdfs:label "renku $schemaVersion".
+                    |  ?agentS rdf:type <http://www.w3.org/ns/prov#SoftwareAgent> .
```

Comment:
would it make sense to use `VALUES` here to allow for both possibilities?

Reply:
Yes, could do that

Reply:
Done